### ISSUE TITLE
Rhino or standalone ?

### ISSUE THREAD

**Issue by BogdanChipara** [alexmojaki/sunhours#2]
Hi,
Interesting plugin. Much more user friendly than Ladybug. It would be nice to have it as Rhino plugin or as a stand-alone software. Any plans in this direction?

Regards

**Comment by alexmojaki** [comment]
No, this project is pretty much unmaintained at this point. I don't even know what Rhino is.

**Comment by BogdanChipara** [comment]
Rhino is used a lot in architecture. It comes with the visual programming interface called grasshopper used for parametric design. 
Website here:
https://www.rhino3d.com/
Plugins page: 
https://www.food4rhino.com/
And Ladybug is the plugin for radiation analysis used inside grasshopper:
https://www.ladybug.tools/

I use ladybug but it's not so userfriendly and it has dependencies. I could also see your software running online.  Users upload the 3d geometry and they get the sun exposure mesh back...
